code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -367,7 +367,7 @@ safeMessage   = dcc "safeMessage" (nounPhraseSP "safe")
   "For the given input parameters, the glass is considered safe."
 sD            = cc' stdOffDist
   (S "the distance from the glazing surface to the centroid of a hemispherical" +:+
-   S "high explosive charge. It is represented by the coordinates" +:+ sParen sdVectorSent)
```

Comment:
why was this changed?

Reply:
I believe this change is from #4007, which this PR builds on.